### ISSUE TITLE
 cannot delete reserved queue

### ISSUE THREAD

**Issue by nutjaa** [jenssegers/laravel-mongodb#752]
[https://github.com/laravel/framework/blob/5.2/src/Illuminate/Queue/DatabaseQueue.php#L242](url)
```

 /**
     * Delete a reserved job from the queue.
     *
     * @param  string  $queue
     * @param  string  $id
     * @return void
     */
    public function deleteReserved($queue, $id)
    {
        $this->database->table($this->table)->where('id', $id)->delete();
    }
```
No id field in jobs collection 
Did i miss some configuration ?

**Comment by giolvani** [comment]
+1

**Comment by davidchchang** [comment]
@nutjaa: you are right, it should be checking for _id instead of id. Pull request (#770) has been submitted and awaiting review.

**Comment by davidchchang** [comment]
I just noticed that @jenssegers has merged this to 2.3 already. I believe this issue can be closed now.
